locales/templates/app.pot

```diff
@@ -0,0 +1,622 @@
+
+msgctxt "geolocation_dialog_block"
+msgid "BLOCK"
+msgstr ""
+
+#. Setting Clear browsing data dialog
+msgctxt "setting_dialog_clear_data"
+msgid "Clear Data"
+msgstr ""
+
+msgctxt "setting_dialog_cancel"
+msgid "Cancel"
+msgstr ""
+
+msgctxt "setting_dialog_browsing_history"
+msgid "Browsing history"
+msgstr ""
+
+msgctxt "setting_dialog_search_history"
+msgid "Search history"
+msgstr ""
+
+msgctxt "setting_dialog_form_history"
+msgid "Form history"
+msgstr ""
+
+msgctxt "setting_dialog_cookies"
+msgid "Cookies"
+msgstr ""
+
+msgctxt "setting_dialog_cache"
+msgid "Cache"
+msgstr ""
+
+#. Download list
+msgctxt "no_downloads"
+msgid "No downloads"
+msgstr ""
+
+msgctxt "download_started"
+msgid "Downloading…"
+msgstr ""
+
+msgctxt "download_file_not_supported"
+msgid "File type not supported"
+msgstr ""
+
+msgctxt "cannot_find_the_file"
+msgid "Cannot find the file."
+msgstr ""
+
+msgctxt "cannot_delete_the_file"
+msgid "Cannot delete the file."
+msgstr ""
+
+#. Download status
+msgctxt "pause"
+msgid "Paused"
+msgstr ""
+
+msgctxt "pending"
+msgid "Pending"
+msgstr ""
+
+msgctxt "running"
+msgid "Downloading"
+msgstr ""
+
+msgctxt "successful"
+msgid "Successful"
+msgstr ""
+
+msgctxt "failed"
+msgid "Failed"
+msgstr ""
+
+msgctxt "unknown"
+msgid "Unknown"
+msgstr ""
+
+#. Setting data saving path dialog
+msgctxt "setting_dialog_sd_card"
+msgid "SD card"
+msgstr ""
+
+msgctxt "setting_dialog_internal_storage"
+msgid "Internal storage"
+msgstr ""
+
+#. BrowsingHistoryActivity
+msgctxt "browsing_history_clear_btn"
+msgid "CLEAR BROWSING HISTORY"
+msgstr ""
+
+msgctxt "browsing_history_empty_view_msg"
+msgid "No history"
+msgstr ""
+
+msgctxt "browsing_history_date_today"
+msgid "Today"
+msgstr ""
+
+msgctxt "browsing_history_date_yesterday"
+msgid "Yesterday"
+msgstr ""
+
+msgctxt "browsing_history_menu_delete"
+msgid "Delete"
+msgstr ""
+
+msgctxt "browsing_history_dialog_confirm_clear_message"
+msgid "Clear all browsing history?"
+msgstr ""
+
+msgctxt "browsing_history_dialog_btn_clear"
+msgid "Clear"
+msgstr ""
+
+msgctxt "remove_from_list"
+msgid "Remove from list"
+msgstr ""
+
+msgctxt "delete_file"
+msgid "Delete file"
+msgstr ""
+
+#. Long Screenshot
+msgctxt "screenshot_saved"
+msgid "Screenshot saved"
+msgstr ""
+
+msgctxt "screenshot_failed"
+msgid "Failed to capture screenshot"
+msgstr ""
+
+msgctxt "screenshot_illustration"
+msgid "capturing screenshot of entire page…"
+msgstr ""
+
+#. Snack bar message
+msgctxt "message_fallback_save_to_primary_external"
+msgid "No SD card detected. File was saved to internal storage."
+msgstr ""
+
+msgctxt "message_start_to_save_to_removable_storage"
+msgid "File saved to SD card"
+msgstr ""
+
+msgctxt "message_removable_storage_space_not_enough"
+msgid "SD card is almost full. File was saved to internal storage."
+msgstr ""
+
+#. ScreenshotGridFragment
+msgctxt "screenshot_grid_empty_text_title"
+msgid "No screenshots"
+msgstr ""
+
+msgctxt "screenshot_grid_empty_text_msg_prefix"
+msgid "Tap "
```

Comment:
Why do you need to concatenate strings here? Use a placeholder and replace it with the code you need.

Reply:
We cannot use the placeholder because it is a button image. We use Spannable to fill in the TextView.

Reply:
Can you clarify? What prevents you from having `Tap {{img}} on the menu to capture an entire page`, replace the placeholder with HTML for the button, and inject that?

Also not completely sure about the \n in the next string.